test/codeu/chat/server/DatabaseTest.java

```diff
@@ -13,26 +16,36 @@
 //Created to test the implementation of the methods in database.Connector.java
 public final class DatabaseTest {
 
+
+  private static final database.Connector con = new database.Connector();
+  private static final RandomUuidGenerator uuidGenerator = new RandomUuidGenerator(
+      Uuids.NULL, System.currentTimeMillis());
+
   @Test
   public void testAddAndDeleteUser() {
     Connector con = new Connector();
     String randomUN = UUID.randomUUID().toString();
     String randomPW = UUID.randomUUID().toString();
-    assertTrue(con.addAccount(randomUN, randomPW));
-    assertTrue(con.deleteAccount(randomUN));
-    randomUN = UUID.randomUUID().toString();
-    randomPW = UUID.randomUUID().toString();
-    assertTrue(con.addAccount(randomUN, randomPW));
-    assertTrue(con.deleteAccount(randomUN));
-    con.closeConnection();
+    assertTrue(
```

Comment:
Stuff like this becomes easier if you have `@Before` and `@After` setUp and tearDown methods :)